toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -926,8 +951,11 @@ PreferencesPopup::PreferencesPopup()
       new CheckBox(tr("Use Default Viewer for Movie Formats"), this);
   CheckBox *minimizeRasterMemoryCB =
       new CheckBox(tr("Minimize Raster Memory Fragmentation *"), this);
-  CheckBox *autoSaveCB = new CheckBox(tr("Save Automatically Every Minutes"));
-  m_minuteFld          = new DVGui::IntLineEdit(this, 15, 1, 60);
+  m_autoSaveCB      = new CheckBox(tr("Save Automatically Every Minutes"));
+  m_autoSaveSceneCB = new CheckBox(tr("Automatically Save the Scene File"));
+  m_autoSaveOtherFilesCB =
+      new CheckBox(tr("Automatically Save Non-Scene Files"));
+  m_minuteFld = new DVGui::IntLineEdit(this, 15, 1, 60);
   CheckBox *replaceAfterSaveLevelAsCB =
```

Comment:
Just like another modification done in the preferences popup ( #664 ), 
how about to use QGroupBox for this option like this?:

<img src="https://cloud.githubusercontent.com/assets/17974955/18705651/fcc27bf6-8029-11e6-9882-61954350ca16.png" width=550>